haystack/nodes/prompt/invocation_layer/hugging_face.py

```diff
@@ -71,6 +71,7 @@ def __init__(
                 "use_fast",
                 "torch_dtype",
                 "device_map",
+                "model_max_length",
```

Comment:
can you please add this param to line 43, supported kwargs?

Reply:
Will do, still working on adding more tests as well.

Reply:
Addressed the commend @agnieszka-m :-)